### PR TITLE
#496 - An inactive logged out user wants to sign up for an offer

### PR DESCRIPTION
**Pull request by magul**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/496
**Reference person:** @Gruszks @magul
**Description:**

An inactive logged out user wants to sign up for an offer

### CODE REVIEW

File: apps/volontulo/views/offers.py [hunk 1]
```diff
@@ -449,6 +449,16 @@ def post(request, slug, id_):  # pylint: disable=unused-argument
         if form.is_valid():
```
Comment (tomaszwozniak) [inline apps/volontulo/views/offers.py:449]: form is_valid() and request.user.is_authenticated() and request.user.is_active

**Comment by tomaszwozniak** [comment]
minor comment

File: apps/volontulo/views/offers.py (hunk 1 above)
Comment (magul) [reply apps/volontulo/views/offers.py:449]: but `request.user.is_authenticated()` and `request.user.is_active` is checked in nested if statements.

See:
- https://github.com/stxnext-csr/volontulo/pull/565/files#diff-9dc4d5e76a83013bca3609b1ec0cad77R450
- https://github.com/stxnext-csr/volontulo/pull/565/files#diff-9dc4d5e76a83013bca3609b1ec0cad77R453

Comment (tomaszwozniak) [reply apps/volontulo/views/offers.py:449]: That is my point to do it in one condition.

**Pull request closed without merging**